### ISSUE TITLE
3.2.x regression with bound map tile requests

### ISSUE THREAD

**Issue by gwacker** [openlayers/ol3#3259]
http://openlayers.org/en/v3.2.0/examples/side-by-side.html?mode=raw

Grab the WebGL map in 3.2.x and drag it until you see new tiles to be requested. Release the panned map (with no panning momentum) and the bound map won't request the new tiles. If there is still panning momentum animation, the tile request fires because of that.

http://openlayers.org/en/v3.1.0/examples/side-by-side.html?mode=raw

This was not an issue in 3.1.x

**Comment by elemoine** [comment]
Thanks for reporting this. I hope we can address it for 3.3.0, which is due next week.

**Comment by ahocevar** [comment]
It is not strictly a regression, but an old bug that is now revealed by changed defaults. Before 3.2.x, the `loadTilesWhileAnimating` and `loadTilesWhileInteracting` map options were `true` by default, and now they are false.

I did some quick research to see what causes the underlying problem, but didn't find anything. So until we find a fix, the workaround is to set at least one of `loadTilesWhileAnimating` or `loadTilesWhileInteracting` to true on the map.

**Comment by igrcic** [comment]
Hi all,

i have also noticed this behaviour when I've updated to 3.2.1 recently. 

I  see that in method `handlePostRender` we resolve number of total tiles loading, based on the view state and `loadTilesWhileAnimating`/`loadTilesWhileInteracting` option parameters. So if there is animation in progress, and `loadTilesWhileAnimating ` is set to false no tile will be loaded until animation is done.

At first I (and my servers) liked the new behavior, but if the animation or interaction -like holding mouse down- last longer, user experience becomes worse as it seems that a map takes longer to load.

I have set `loadTilesWhileAnimating` parameter as Andrea has suggested, and things are OK again. 

One question Andrea: have you maybe thought of exporting `maxTotalLoadingWhileAnimating` as a user definable parameter? Or you think 8 is a best option?

Gruss,
Ivan
